tools/osbuilder/rootfs-builder/rootfs.sh

```diff
@@ -669,6 +669,11 @@ EOF
 			info "Adding agent config for ${AA_KBC}"
 			AA_KBC_PARAMS="offline_sev_kbc::null" envsubst < "${script_dir}/agent-config.toml.in" | tee "${ROOTFS_DIR}/etc/agent-config.toml"
 		fi
+		if [ "${AA_KBC}" == "online_sev_kbc" ]; then
+			info "Adding agent config for ${AA_KBC}"
+			#KBC URI will be specified in the config file via kernel params
+			AA_KBC_PARAMS="online_sev_kbc::172.16.17.43:44444" envsubst < "${script_dir}/agent-config.toml.in" | tee "${ROOTFS_DIR}/etc/agent-config.toml"
```

Comment:
Is there a way to avoid having to rebuild the image for a different KBC URL?

Reply:
This is just a default value. The agent-config can now be overwritten by the kernel command line. So you can update the kbc_params by updating the kata config file. We will also soon allow people to specify that kata config value via a kubernetes annotation to make it even more flexible.

Reply:
@Alex-Carter01 you might want to swap out that IP address for something that looks more like a default and make the comment a bit more clear.

Reply:
Thanks, for the clarification, @fitzthum. Can the identity of the trusted KBC (e.g its public key) be also specified in AA_KBC_PATAMS? You'd mentioned in another issue that the identity of the KBC is instead expected to be enforced via ability to obtain a secret from it.

Reply:
Well we have some size limitations for the kernel params so that might not be totally practical. I blog post about this is all written up I"m just haggling with management about where I can post it. Hopefully will be out this week.